loopchain/baseservice/node_subscriber.py

```diff
@@ -125,7 +125,7 @@ def __del__(self):
             vote = BlockVotes.deserialize_votes(votes_serialized)
         else:
             vote = None
-        blockchain = ObjectManager().channel_service.block_manager.get_blockchain()
+        blockchain = ObjectManager().channel_service.block_manager.blockchain
```

Comment:
How about accessing `BlockManager.blockchain' variable as `@property?`, not directly accessing it?